css/properties/border-bottom-left-radius.json

```diff
@@ -14,6 +14,9 @@
                 "version_added": "1"
               }
             ],
+            "chrome_android": {
+              "version_added": true
```

Comment:
Should be `"18"`.

Reply:
thanks - I've just made a commit added this and the others.